wallet/AndroidManifest.xml

```diff
@@ -2,8 +2,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="de.schildbach.wallet_test"
     android:installLocation="internalOnly"
-    android:versionCode="60106"
-    android:versionName="6.1.2">
+    android:versionCode="60200"
```

Comment:
I like keep version bumps in separate commits so its easy to tell from the history what commits done in each version.
![version-commit](https://user-images.githubusercontent.com/5956129/47832721-40dcbe80-dd54-11e8-9a1b-8d3b726f910e.PNG)

Reply:
Commit #1b36c77 reverts version bump and fixes a crash when service is started by alarm.